hb_report/utillib.py

```diff
@@ -1663,6 +1663,15 @@ def lsof_ocfs2_device():
 def dump_ocfs2():
     ocfs2_f = os.path.join(constants.WORKDIR, constants.OCFS2_F)
     with open(ocfs2_f, "w") as f:
+        rc, out, err = crmutils.get_stdout_stderr("mounted.ocfs2 -d")
+        if rc != 0:
```

Comment:
How is this check working? I mean, what happens if this returns an error.
Should the code just continue?

Reply:
If running this command failed, error message will be recorded in `ocfs2.txt` then this function return, other functions not related with ocfs2 dumping continued